### PR TITLE
docs(readme): [PFMT-3877] PHP Laravel Test README

### PR DESCRIPTION
**Pull request by isudnop**
### What problem does this PR solve?
Missing PHP challenge readme file for candidates

### How does this PR solve it?
Add README.md file for PHP Laravel test (modified from NodeJs one)

### CODE REVIEW

**Comment by pmlo-sirisak-l** [comment]
Let's add a link to the main README.md file for the PHP submission.

**Comment by isudnop** [comment]
Linked to main README

File: challenge-php-laravel/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Pomelo PHP Laravel Challenge
+
+Welcome to the Pomelo PHP Laravel challenge!
+
+## Principles & Ground Rules
+
+Please focus on those four principles while writing code: **clarity**, **simplicity**, **defensiveness** and  **resilient**.
```
Comment (pmlo-sirisak-l) [inline challenge-php-laravel/README.md]:
```suggestion
Please focus on those four principles while writing code: **clarity**, **simplicity**, **defensiveness** and  **resilience**.
```

**Review by pmlo-sirisak-l — APPROVED** [review]
lgtm

**Pull request merged**